socorro/external/rabbitmq/crashstorage.py

```diff
@@ -299,3 +303,33 @@ class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
         False
     )
 
+
+#==============================================================================
+class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
+    required_config = Namespace()
+    required_config.rabbitmq_class = change_default(
+        RabbitMQCrashStorage,
+        'rabbitmq_class',
+        ConnectionContext,
+    )
+    required_config.routing_key = change_default(
+        RabbitMQCrashStorage,
+        'routing_key',
+        'socorro.reprocessing'
+    )
+
+    def __init__(self, config, quit_check_callback=None):
+        config.forbidden_keys = ''
+        config.redactor_class = Redactor
+        config.throttle = 100
+        config.filter_on_legacy_processing = False  # must be set
+        super(ReprocessingRabbitMQCrashStore, self).__init__(
+            config, quit_check_callback=quit_check_callback
+        )
+
+    def post(self, crash_id):
```

Comment:
I don't like it. ``crashstorage`` classes should never be directly exposed in the API. In the new world of separate repos, this will be a part of ``socorrolib``, not of ``webapp`` or ``middleware``.